clients/admin-ui/src/features/taxonomy/TaxonomyTabs.tsx

```diff
@@ -0,0 +1,21 @@
+import { Box } from "@fidesui/react";
+
+import DataTabs from "../common/DataTabs";
+import DataCategoriesTab from "./DataCategoriesTab";
+import DataSubjectsTab from "./DataSubjectsTab";
+import DataUsesTab from "./DataUsesTab";
+import IdentifiabilityTab from "./IdentifiabilityTab";
+
+const TABS = [
+  { label: "Data Categories", content: <DataCategoriesTab /> },
```

Comment:
The engineer in me thinks that this could probably be made programmatic? along with the 4 files that generate the tabular view. But I don't know enough about front-end to know if that's a bad paradigm here or not

Reply:
yeah, I think these will eventually start looking different, or they can be consolidated to one file. I had trouble deciding right now while pieces aren't built yet, so just went with what is easiest for now. but you've definitely got the right instinct!